lib/tournament.json

```diff
@@ -625,6 +625,44 @@
       }
     ]
   ],
+  "rankings": [
```

Comment:
Wonder if for the sake of simplicity in the FE, would combining all the information regarding a team together with the team declaration be easier.  This way I don't need to do mappings of teams to extra data.

So essentially two structures...teams/round-robin.  The teams part contains the name, members, rankings, etc.  and round-robin shows the upcoming matches and whether they're completed or not?